ui/app/templates/components/namespace-link.hbs

```diff
@@ -1,16 +1,18 @@
-{{#link-to "vault.cluster.secrets" (query-params namespace=normalizedNamespace)
-  class=(concat "is-block " class)
-}}
+<LinkTo
+  @params={{array "vault.cluster.secrets" (query-params namespace=normalizedNamespace)}}
+  class={{concat "is-block " class}}
+  data-test-namespace-link={{this.normalizedNamespace}}
```

Comment:
why do we need the `this.` now?

Reply:
It's not required, but it's helpful to clarify what's state on the component, what's an argument to a component (`@foo`), and what's a helper. https://github.com/emberjs/rfcs/blob/master/text/0276-named-args.md has more about that. I should probably update the other use of `normalizedNamespace` too haha.